Test/Test/CharOccurrences.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+
+class CharOccurrences {
+	private:
+		const int static possibleChars = 128;
+		int charOccurs[possibleChars] = {};
+	public:
+		static int getPossibleChars();
+		void calculateOccurs(int charOccurs[], std::string const &inputLine);
+		bool calculateOccursInFile(std::string const &inputFilepath, int charOccurs[]);
+		bool outputToFile(std::string const &outputFilePath, int charOccurs[]);
+		int * getCharOccurs();
```

Comment:
Do we allow to change content for external users of our class?
When we return a pointer to the array anyone can change the data kept in it. At least we can ask it to be a const (for sure it is still changeable, but at least we mark "DO NOT CHANGE IT, KEEP YOUR HANDS AWAY")

Reply:
It seems that this method is just useless, and was a byproduct of me moving those functions from main into class and then not bothering to change much. Like, why even pass output array as an argument?
Now this class keeps the charOccurs array to itself and doesn't return it anywhere. The only way for a user to "access" it is to use outputToFile.
That seems more reasonable, doesn't it? I hope that I didn't make it even worse somehow :P